aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -147,29 +225,46 @@ module aptos_framework::staking_config {
         staking_config.recurring_lockup_duration_secs = new_recurring_lockup_duration_secs;
     }
 
+    public fun update_rewards_rate(
+        _aptos_framework: &signer,
+        _new_rewards_rate: u64,
+        _new_rewards_rate_denominator: u64,
+    ) {
+        assert!(false, EDEPRECATED_FUNCTION);
```

Comment:
I think as a precaution, we might not want to deprecate this in case we need to change the old per-epoch rate for some reason. Wdyt?